Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -171,7 +173,21 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
   switch (req.command)
   {
   case GO_IDLE_STATE:
-    // libogc can use it during init..
+    INFO_LOG(IOS_SD, "GO_IDLE_STATE");
+    if (m_Card)
+    {
+      if (m_Card.GetSize() > SDHC_BYTES)
+      {
+        m_Status |= CARD_SDHC;
+        Memory::Write_U32(0xc0ff8000, _BufferOut);
+      }
+      else
+      {
+        Memory::Write_U32(0x80ff8000, _BufferOut);
+        // No further initialization required.
+        m_Status |= CARD_INITIALIZED;
```

Comment:
Somewhat related to the previous comment, but if you were to use `m_Status` (...if thats possible) and swap the statements around, you could move the `Write_U32` to the end and only call it once.